playwright/pages/blocks-dashboard.page.ts

```diff
@@ -1,15 +1,252 @@
 import { expect, Page } from "@playwright/test";
+import moment from "moment";
+
+import { BlockInformationDto } from "playwright/api/dtos/blockInformation.dto";
 
 export function blocksDashboard(page: Page) {
   const blocksTable = page.getByTestId("blocks-card");
   const firstBlocksID = page.locator("td > [aria-label]").first();
 
+  const sidebarBlockchainButton = page.getByTestId("menu-button-blockchain");
+  const epochsTab = page.getByTestId("submenu-button-blocks");
+  const searchBarEpoch = page.getByTestId("all-filters-dropdown");
+  const deatailPageTitle = page.getByTestId("detail.page.title");
+  const addressDetailTitle = page.getByTestId("address-detail-title");
+
+  //Blocks Table
+  const blocksTableTitleBlock = page.getByTestId("blocks.table.title.block");
+  const blocksTableValueBlock = page.getByTestId("blocks.table.value.block#2");
+  const blocksTableTitleBlockId = page.getByTestId("blocks.table.title.blockId");
+  const blocksTableValueBlockId = page.getByTestId("blocks.table.value.blockId#2");
+  const blocksTableTitleEpoch = page.getByTestId("blocks.table.title.epoch");
+  const blocksTableValueEpoch = page.getByTestId("blocks.table.value.epoch#2");
+  const blocksTableTitleSlot = page.getByTestId("blocks.table.title.slot");
+  const blocksTableTitleAbsSlot = page.getByTestId("blocks.table.title.absoluteSlot");
+  const blocksTableTitleCreateAt = page.getByTestId("blocks.table.title.createAt");
+  const blocksTableTitleTransactionsCount = page.getByTestId("blocks.table.title.transactions");
+  const blocksTableTitleFee = page.getByTestId("blocks.table.title.fee");
+  const blocksTableTitleOutput = page.getByTestId("blocks.table.title.output");
+  const blocksTableValueSlot = page.getByTestId("blocks.table.value.slot#2");
+
+  //Block detail
```

Comment:
As I mentioned in the epochs test this block detail page locators and respective functions that are executed over the block detail page should be in the block-detail.page.ts class